### PR TITLE
CONTRIBUTING.md - Don't rebase always on master

### PR DESCRIPTION
**Pull request by SpacePossum**
I find the suggestion to rebase with master misleading/confusing

### CODE REVIEW

**Comment by sstok** [comment]
Newcomers will still want to use know what branch to rebase on, when I got started with git I always rebased on my current branch :stuck_out_tongue_closed_eyes:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 You can do some things to increase the chance that your pull request is accepted the first time:
 
  * Submit one pull request per fix or feature.
- * If your changes are not up to date - rebase your branch on master.
+ * If your changes are not up to date - rebase your branch.
```
Comment (keradus) [inline CONTRIBUTING.md]: `on your parent branch?`

Comment (GrahamCampbell) [inline CONTRIBUTING.md]: yeh, parent makes more sense

Comment (GrahamCampbell) [inline CONTRIBUTING.md]: rather than origin, lol

Comment (SpacePossum) [inline CONTRIBUTING.md]: :+1: done  :)
@sstok I always rebased on some local verion of master, that was fun as well :cold_sweat: :blush:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,7 +20,7 @@
 You can do some things to increase the chance that your pull request is accepted the first time:
 
  * Submit one pull request per fix or feature.
- * If your changes are not up to date - rebase your branch on master.
+ * If your changes are not up to date - rebase your branch n your parent branch.
```
Comment (keradus) [inline CONTRIBUTING.md]: typo (`n` -> `on`)

Comment (SpacePossum) [inline CONTRIBUTING.md]: d'oh

**Comment by keradus** [comment]
:+1:

**Comment by gharlan** [comment]
:+1:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,7 +20,7 @@
 You can do some things to increase the chance that your pull request is accepted the first time:
 
  * Submit one pull request per fix or feature.
- * If your changes are not up to date - rebase your branch on master.
+ * If your changes are not up to date - rebase your branch on your parent branch.
```
Comment (GrahamCampbell) [inline CONTRIBUTING.md]: :-1: for "your parent". "the parent" makes more sense.

Comment (SpacePossum) [inline CONTRIBUTING.md]: I'm following https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1609#discussion_r48011706

Comment (GrahamCampbell) [inline CONTRIBUTING.md]: I still disagree though. It's not `your parent`. It's just `the parent`.

Comment (keradus) [inline CONTRIBUTING.md]: :+1: for `the`

Comment (keradus) [inline CONTRIBUTING.md]: I'm not almighty @SpacePossum ;) I could be wrong ;)

Comment (SpacePossum) [inline CONTRIBUTING.md]: `the` it is  ^-^

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**